server/plugin/webhook.go

```diff
@@ -702,6 +702,13 @@ func (p *Plugin) postIssueCommentEvent(event *github.IssueCommentEvent) {
 		Type:   "custom_git_comment",
 	}
 
+	repoName := repo.GetFullName()
```

Comment:
0/5 collapse into `repoName := strings.ToLower(repo.GetFullName())` for readability.